profiles/amity-ak97.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/51391473/173429345-e8914595-0281-4f1b-bded-afea0f6db842.png",
```

Comment:
I recommend more of an image like this. What do you think? 🙂

![image](https://user-images.githubusercontent.com/51391473/183324033-a83244ff-f987-4bf1-a792-9965b6c20a2d.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/183324033-a83244ff-f987-4bf1-a792-9965b6c20a2d.png",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question